pkg/assembler/backends/testing/pkg.go

```diff
@@ -17,206 +17,405 @@ package testing
-	client.registerPackage("conan", "", "openssl", "3.0.3", "")
-	// pkg:deb/debian/attr@1:2.4.47-2?arch=amd64
-	client.registerPackage("deb", "debian", "attr", "1:2.4.47-2", "", "arch", "amd64")
-	// pkg:deb/debian/attr@1:2.4.47-2?arch=source
-	client.registerPackage("deb", "debian", "attr", "1:2.4.47-2", "", "arch", "source")
-	// pkg:deb/debian/curl@7.50.3-1?arch=i386&distro=jessie
-	client.registerPackage("deb", "debian", "curl", "7.50.3-1", "", "arch", "i386", "distro", "jessie")
-	// pkg:deb/debian/dpkg@1.19.0.4?arch=amd64&distro=stretch
-	client.registerPackage("deb", "debian", "dpkg", "1.19.0.4", "", "arch", "amd64", "distro", "stretch")
-	// pkg:deb/ubuntu/dpkg@1.19.0.4?arch=amd64
-	client.registerPackage("deb", "ubuntu", "dpkg", "1.19.0.4", "", "arch", "amd64")
-	// pkg:docker/cassandra@latest
-	client.registerPackage("docker", "", "cassandra", "latest", "")
-	// pkg:docker/cassandra@sha256:244fd47e07d1004f0aed9c
-	client.registerPackage("docker", "", "cassandra", "sha256:244fd47e07d1004f0aed9c", "")
-	// pkg:docker/customer/dockerimage@sha256:244fd47e07d1004f0aed9c?repository_url=gcr.io
-	client.registerPackage("docker", "customer", "dockerimage", "sha256:244fd47e07d1004f0aed9c", "", "repository_url", "gcr.io")
-	// pkg:docker/smartentry/debian@dc437cc87d10
-	client.registerPackage("docker", "smartentry", "debian", "dc437cc87d10", "")
-	// pkg:generic/bitwarderl?vcs_url=git%2Bhttps://git.fsfe.org/dxtr/bitwarderl%40cc55108da32
-	client.registerPackage("generic", "", "bitwarderl", "", "", "vcs_url", "git%2Bhttps://git.fsfe.org/dxtr/bitwarderl%40cc55108da32")
-	// pkg:generic/openssl@1.1.10g
-	client.registerPackage("generic", "", "openssl", "1.1.10g", "")
-	// pkg:generic/openssl@1.1.10g?download_url=https://openssl.org/source/openssl-1.1.0g.tar.gz&checksum=sha256:de4d501267da
-	client.registerPackage("generic", "", "openssl", "1.1.10g", "", "download_url", "https://openssl.org/source/openssl-1.1.0g.tar.gz", "checksum", "sha256:de4d501267da")
-	// pkg:oci/debian@sha256:244fd47e07d10?repository_url=ghcr.io/debian&tag=bullseye
-	client.registerPackage("oci", "", "debian", "sha256:244fd47e07d10", "", "repository_url", "ghcr.io/debian", "tag", "bullseye")
-	// pkg:oci/hello-wasm@sha256:244fd47e07d10?tag=v1
-	client.registerPackage("oci", "", "hello-wasm", "sha256:244fd47e07d10", "", "tag", "v1")
-	// pkg:oci/static@sha256:244fd47e07d10?repository_url=gcr.io/distroless/static&tag=latest
-	client.registerPackage("oci", "", "static", "sha256:244fd47e07d10", "", "repository_url", "gcr.io/distroless/static", "tag", "latest")
-	// pkg:pypi/django-allauth@12.23
-	client.registerPackage("pypi", "", "django-allauth", "12.23", "")
-	// pkg:pypi/django@1.11.1
-	client.registerPackage("pypi", "", "django", "1.11.1", "")
-	// pkg:pypi/django@1.11.1#subpath
-	client.registerPackage("pypi", "", "django", "1.11.1", "subpath")
-	// pkg:pypi/kubetest@0.9.5
-	client.registerPackage("pypi", "", "kubetest", "0.9.5", "")
-}
+	ctx := context.Background()
```

Comment:
I think we can fully delete registerAllPackages?

Reply:
YESSS! lol

Reply:
I am going to put a todo here and instead of deleting, add it to the unit tests